player.go

```diff
@@ -69,26 +69,44 @@ func (a *API) GetDevices() ([]*Device, error) {
 
 // Play starts a new context or resume current playback on the user's active device.
 // https://developer.spotify.com/documentation/web-api/reference/#endpoint-start-a-users-playback
-func (a *API) Play(deviceID string, uris ...string) error {
+// contextURI added to allow for album and playlist support - A blank context uri will play just the track uri.
+func (a *API) Play(deviceID, contextURI string, uris ...string) error {
 	v := make(url.Values)
 	if deviceID != "" {
 		v.Add("device_id", deviceID)
 	}
 
-	if len(uris) == 0 {
+	if (len(uris) == 0) && (len(contextURI) == 0) {
 		return a.put("v1", "/me/player/play", v, nil)
 	}
```

Comment:
I don't think we need this. If a user passes both of these options they should get an error, since it's an illegal combination.
```suggestion
```

Reply:
This was logic from before I looked at this file that I just moved around :) I will remove it.